Resources/Documentation/dv_info.md

```diff
@@ -1,32 +1,126 @@
 # DV Capture
 
-To capture DV formats in vrecord (when installed in macOS), you will need to have your DV deck connected directly to your computer via FireWire input. Make sure your deck is in “Local” mode.
+**macOS:** To capture DV formats (DV, DVCAM, MiniDV, DVCPRO and Digital8) in vrecord, you will need to have your DV deck connected directly to your computer via <a href="https://mipops.github.io/dvrescue/sections/dv_transfer_station.html" target="_blank">FireWire input</a>. Make sure your deck is turned on and in “Remote” mode. Additional settings for your DV device may apply to ensure a successful connection. To view manuals and settings notes for specific decks, players and camcorders, please see the <a href="https://mipops.github.io/dvrescue/sections/deck_guide.html" target="_blank">DVRescue Deck Guide</a>. You may also need to adjust your  <a href="https://mipops.github.io/dvrescue/sections/troubleshooting.html" target="_blank">privacy settings</a> and allow each DV device to connect prior to use with vrecord.
+
+> **_NOTE:_** vrecord uses the <a href="https://github.com/mipops/dvrescue" target="_blank">DVRescue toolkit</a>  to facilitate dvcapture. HDV tapes are not in the current scope of this project and cannot be captured using DVRescue or vrecord. For more information, please see this <a href="https://github.com/mipops/dvrescue/issues/53" target="_blank">issue</a> discussing the differences between DV and HDV. 
 
 ## DV Settings
 
-1. To select the deck you would like to use, go into the configuration mode either by:
-   - run `vrecord -e`
-   - clicking on the "Edit Settings" button in the GUI (run `vrecord`)
+1. Open vrecord's configuration mode via one of the following options:
+   - run `vrecord -e` to open the settings menu in the GUI.
+   - run `vrecord` and click on the "Edit Settings" button in the GUI.
 
 ![Decklink](../dv_vrecord_decklink.jpg "DV Capture") 
 
-1. Switch from the "Decklink" tab to the "DV" tab under "Input Options" at the top of the configuration window.
```

Comment:
I think simplifying language to just say `Select the "DV" tab under "Input Options" at the top of the configuration window` is more clear (since we have multiple tabs at this point)